core/src/main/java/org/apache/accumulo/core/util/cache/Caches.java

```diff
@@ -75,22 +78,26 @@ public void registerMetrics(MeterRegistry registry) {
     this.registry = registry;
   }
 
-  private void setupMicrometerMetrics(Caffeine<Object,Object> cacheBuilder, String name) {
+  private boolean setupMicrometerMetrics(Caffeine<Object,Object> cacheBuilder, String name) {
     if (registry != null) {
       try {
         cacheBuilder.recordStats(
             () -> new CaffeineStatsCounter(registry, name, MetricsUtil.getCommonTags()));
+        return true;
       } catch (IllegalStateException e) {
         // recordStats was already called by the cacheBuilder.
```

Comment:
Would be good to log this in case its not the expected case you want to ignore.  Could log it at trace.

Reply:
Logging added in 8bee224.